guides/common/modules/proc_setting-up-pull-client.adoc

```diff
@@ -0,0 +1,34 @@
+[id="setting-up-pull-client_{context}"]
+= Setting Up the Remote Execution Pull Client
+
+For {SmartProxies} in `pull-mqtt` mode, hosts can subscribe to remote jobs using the remote execution pull client.
+No SSH connection is required.
+
+Use this procedure to configure a host to use the pull client:
+
+.Prerequisites
+* The host is registered to {ProjectName}.
+* The host's {SmartProxy} is in `pull-mqtt` mode.
+* The {Team} {project-client-name} repository is enabled and synchronized on {ProjectServer}, and enabled on the host.
+* The host is able to communicate with its {SmartProxy} over MQTT using port 1883.
+* The host is able to communicate with its {SmartProxy} over HTTPS.
+
+.Procedure
+. On the host, install the `katello-pull-transport-migrate` RPM package:
++
+----
+# yum install katello-pull-transport-migrate
+----
+The host's `subscription-manager` configuration and consumer certificates are used to configure the `yggdrasil` client on the host, and the worker is started.
```

Comment:
"the worker" -> "the yggdrasild worker" or "pull mode client worker"? (we speak about 'worker' for the first time)